modes/magit/README.org

```diff
@@ -0,0 +1,215 @@
+  10. When =evil-collection-magit-use-y-for-yank= is non nil, =C-w= will prefix the evil
+      window switching functions from magit buffers.
+  11. =evil-collection-magit-use-y-for-yank= is now the default. It has worked well for me so
+      far, and I've had good feedback, but please let me know if you see issues.
+      You can use the original behavior with =(setq evil-collection-magit-use-y-for-yank
+      nil)=. See the table below for a summary of differences.
+
+
+* Installation and Use
+
+Everything is contained in evil-collection-magit.el, so you may download and load that file
+directly. The recommended method is to use MELPA via package.el (=M-x
+package-install RET evil-collection-magit RET=).
+
+Evil and Magit are both required. After requiring those packages, the following
+will setup the new key bindings for you.
+
+#+BEGIN_SRC emacs-lisp
+;; optional: this is the evil state that evil-collection-magit will use
+;; (setq evil-collection-magit-state 'normal)
+;; optional: disable additional bindings for yanking text
+;; (setq evil-collection-magit-use-y-for-yank nil)
+(require 'evil-collection-magit)
+#+END_SRC emacs-lisp
+
+** Note on evil usage
+
+This package assumes that you either use the global variant of evil mode (e.g.,
+through =(evil-mode 1)=), or at least have =evil-local-mode= (the local variant)
+enabled in the magit buffers you want these bindings to take effect in. When
+evil is disabled in a magit buffer, this package will not affect the default key
+bindings (with one minor exception).
+
+** Note on =evil-collection-magit-use-y-for-yank=
+
+=evil-collection-magit-use-y-for-yank= enables evil's visual state for linewise selection,
+and as a consequnce =y= will yank text from the buffer.
+
+With this enabled which it is by default evil-collection-magit uses =v= and =V= to select
+by line. Selection in magit occurs linewise, so this choice is to avoid
+confusion that might arise if someone thought they could stage part of a line
+with =v= for example.
+
+** Text mode
+
+Text mode can be toggled with =evil-collection-magit-toggle-text-mode= (triggered with
+=C-t= or =\=). This takes nearly any magit buffer out of the related magit mode
+and puts it into =text-mode=. This allows free movement in the buffer using the
+standard evil movement and selection commands, making it easy to for example
+copy arbitrary text in the buffer. It also effectively prevents magit keys from
+shadowing evil ones, so =f= runs =evil-find-char= instead of
+=magit-fetch-popup=, allowing all vim related movement commands to be used in
+magit buffers. You can think of this if you like as another state for evil-collection-magit
+to be in.
+
+Several requests have been made to allow selecting and copying arbitrary text in
+the magit buffers, but there are many conflicts between evil bindings and magit
+bindings and there is no elegant solution to this problem in my opinion. Text
+mode is the best that I have come up with.
+
+* Key Bindings
```

Comment:
No titlecase for consistency with other sections.

Reply:
I think it's already consistent unless I'm misunderstanding.